app/forms/RemoveSupportingDocumentFormProvider.scala

```diff
@@ -19,18 +19,13 @@ package forms
 import javax.inject.Inject
 
 import play.api.data.Form
-import play.api.data.Forms.set
 
 import forms.mappings.Mappings
-import models.RequiredInformation
 
-class RequiredInformationFormProvider @Inject() extends Mappings {
+class RemoveSupportingDocumentFormProvider @Inject() extends Mappings {
 
-  def apply(): Form[Set[RequiredInformation]] =
+  def apply(): Form[Boolean] =
```

Comment:
I believe this is also something that will go away on a rebase too. This was probably not changed in this PR and just reflects changes on main (in a strange way)